CONTRIBUTING.md

```diff
@@ -109,12 +109,14 @@ CSS
 
 ```
 
-Here is a full list of the available CSS variables:
-
-- `--darkreader-neutral-background` should be mostly used for elements that have a wrong background color (usually bright backgrounds that should be dark).
-- `--darkreader-neutral-text` should be used for elements with a wrong text color (usually dark texts that should be light).
-- `--darkreader-selection-background` corresponds to user's Selection Background Color setting.
-- `--darkreader-selection-text` corresponds to user's Selection Text Color setting.
+Here is a full table of available CSS variables:
+
+| Variable                            	| Description                     	| Use                                                         	|
+|-------------------------------------	|---------------------------------	|-------------------------------------------------------------	|
+| `--darkreader-neutral-background`   	| Background Color                	| Mostly used for elements that have a wrong background color 	|
```

Comment:
I think the description could be somewhat longer. Something along the lines of `The neutral background color that correspondents to the user's settings`. The same with the text color.

Reply:
>  Something along the lines of `The neutral background color that correspondents to the user's settings.`


*Causally copy pastes*

Reply:
Done
<img width="1035" alt="image" src="https://user-images.githubusercontent.com/66189242/110687235-32cac280-81d8-11eb-83c9-e2762d3302e5.png">